tests/unit/Interactions/Internal/WebDriverCoordinatesTest.php

```diff
@@ -15,10 +16,19 @@ public function testConstruct()
 
         $webDriverCoordinates = new WebDriverCoordinates(null, $in_view_port, $on_page, 'auxiliary');
```

Comment:
This test does seem to focus on internal implementation... I know its unrelated to PHP 8, but instead of using reflectioins, we can rather replace it with something more useful, so I suggest replacing  whole method it with following:

```php
public function testConstruct()
    {
        $in_view_port = function () {
            return new WebDriverPoint(0, 0);
        };
        $on_page = function () {
            return new WebDriverPoint(10, 10);
        };

        $webDriverCoordinates = new WebDriverCoordinates(null, $in_view_port, $on_page, 'auxiliary');

        $this->assertEquals(new WebDriverPoint(0, 0), $webDriverCoordinates->inViewPort());
        $this->assertEquals(new WebDriverPoint(10, 10), $webDriverCoordinates->onPage());
        $this->assertSame('auxiliary', $webDriverCoordinates->getAuxiliary());
    }
```

Then we can also drop following "testGetAuxiliary" method.

Reply:
Done 👍